### DISCUSSION TITLE
Custom FBW spoilers with no effect

### DISCUSSION THREAD

**Discussion by CptBayer** [flybywiresim/aircraft#4606] [General]
Hi developers,

since the last udate of the custom FBW mod, the spoilers have no effect! During descent i tried to reduce speed from 290 kts to 250 kts ... nothing happens .... no deccelaration ... Nor with a vertical speed of 2000 ft, nor with a vs of 1500 ft, nor with a cs of 1000 ft ... I tried as well full spoilers out, which is in general not usable in flight... no effect ...

Speed trend arrow is also gone .... 

Greets Klaus

**Comment by ExampleWasTaken** [comment]
They still work but they now work realistically. When the AP is engaged they only deploy 50%.

Here's the FCOM entry about the various conditions. Not all of them are necessarily implemented yet, but just to give you an idea:
![image](https://user-images.githubusercontent.com/58574351/116142316-0f050280-a6da-11eb-995e-66897ffccecb.png)
